Omnibus.md

```diff
@@ -2759,34 +2778,35 @@ uses drama and theatrics has a better chance of success.
 ### Embrace Death (6th Circle) ### {#embrace-death}
 
 __Uses:__ 1 - __Verbal:__ 40 words - __Material:__ A container at least 4 inches
-in diameter able to hold the spellcaster's scalp token[, labeled with
+in diameter able to hold the spellcaster's [soul]{.new} token[, labeled with
```

Comment:
Okay, even though the old text was inconsistent about "scalp" vs. "scalp token", and this specific instance already had the word "token", I think "soul token" is the thing being added by this proposal, so I'd like to see "token" appear as new.

Reply:
The 2018 Omnibus uses the phrase "scalp token" all of fives times.  Four times in this spell and once in Regenerate the Scalp.  I'd like all five of those instances of "token" to be marked as new, but I'll concede if the OEC's majority disagrees.

Reply:
I don't really care, but technically token should be black. I'll side with the third person to weigh in.

Reply:
It depends.  If we are using red to mark purely textual changes, then I think token should not be included (as the words were already there).

If we are saying that the phrase "soul token" is replacing the phrase "scalp token," I could see including token.  However, I think that opens another can of worms in having to reevaluate if other proposals change the words around them meaningfully enough to warrant including words that were already there as new text. I'm not necessarily against that, but it does seem a bit much to me.

I'm fine with whatever we do as long as we are consistent with implementing it.  I'm going to leave things as is for now, but may change it if we agree that the second case is how we want to handle things.

Reply:
Personally, I don't see the need the mark "token" as new (in the cases of scalp token --> soul token).  To me, the new marker is to assist in identifying actual changes in the text.  Marking the whole phrase could mislead people into thinking the word "token" didn't exist in the first place.